cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -48,7 +48,7 @@ use polkadot_parachain_primitives::primitives::RelayChainBlockNumber;
 use polkadot_runtime_parachains::FeeTracker;
 use scale_info::TypeInfo;
 use sp_runtime::{
-	traits::{Block as BlockT, BlockNumberProvider, Hash},
+	traits::{Block as BlockT, BlockNumberProvider, Hash, One},
```

Comment:
```suggestion
	traits::{Block as BlockT, BlockNumberProvider, Hash},
```
 Clippy https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4853813